vllm/worker/model_runner.py

```diff
@@ -683,6 +707,18 @@ def list_loras(self) -> Set[int]:
 
     @torch.inference_mode()
     def capture_model(self, kv_caches: List[KVCache]) -> None:
+        """Cuda graph capture a model.
+
+        Note that cuda graph performance gain is negligible if number
+        of batched tokens are less than 200. And since Cuda graph
+        requires fixed sized tensors, supporting large/variable batch
+        size requires high GPU memory overhead. Thus, vLLM only captures
+        decoding requests. Mixed batch (chunked prefill + decoding) or
+        prefill requests are not captured.
+
+        this API assumes the shape is N batches of tokens flattened to 1D
+        tensor, where is token's seqlen is 1.
```

Comment:
I actually didn't quite understand what you mean by `where is token's seqlen is 1`. Could you elaborate more?